modules/transactionpool/accept.go

```diff
@@ -306,6 +337,30 @@ func (tp *TransactionPool) acceptTransactionSet(ts []types.Transaction, txnFn fu
 		}
 	}
 
+	// Create the UnconfirmedTransactionSet object for this transaction set, and
+	// then add it to a diff for the subscribers.
+	ids := make([]types.TransactionID, 0, len(ts))
+	sizes := make([]uint64, 0, len(ts))
+	for i := range ts {
+		sizes = append(sizes, uint64(ts[i].MarshalSiaSize()))
+		ids = append(ids, ts[i].ID())
+	}
+	ut := &modules.UnconfirmedTransactionSet{
+		Change: &cc,
+		ID:     modules.TransactionSetID(setID),
+
+		IDs:          ids,
+		Sizes:        sizes,
+		Transactions: ts,
+	}
+	// Add this diff to our set of subscriber diffs.
+	tp.subscriberSets[setID] = ut
+	diff := new(modules.TransactionPoolDiff)
+	diff.AppliedTransactions = append(diff.AppliedTransactions, ut)
+	for _, subscriber := range tp.subscribers {
+		subscriber.ReceiveUpdatedUnconfirmedTransactions(diff)
+	}
+
```

Comment:
This looks good to me. The only thing that is bothering me a little is, that `handleConflicts` implicitly accepts the tset using almost the same code. Maybe we could simplify the code by checking for conflicts before checking the composition, fees etc. in `acceptTransactionSet`. 
If there are no conflicts we continue with `ts` and if there were,`handleConflicts` creates and returns the `superset` which is then checked and used to create the diff instead of `ts`.